semgrep_output_v1.atd

```diff
@@ -909,8 +909,8 @@ type tag = string
 
 (* Response by the backend to the CLI to the POST /scans/<scan_id>/config *)
 type scan_config = {
```

Comment:
So I guess we can delete this type then in favor of your new 'scan_configuration'

Reply:
Yes, if it's not used at all (which seems likely) then we can remove this, I'd vote for doing this in a separate PR though.